data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1924 @@
+		has "Heliarch Drills 2-C: done"
+		not "Lunarium: Questions: done"
+	on offer
+		conversation
+			`Lotuora is speaking with a Kimek Heliarch captain at the entrance to the spaceport. "Ah, there the Captain is," she tells the Kimek. "Captain <last>, your sparring partner, this is," she says, gesturing to the Kimek, who salutes you with a short bow of her head. "Excited to test her ship against an outsider, she is."`
+			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, where she boards a Punisher and flies away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall."`
+			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
+				accept
+	npc disable save
+		government "Heliarch Test Dummy"
+		personality disables staying nemesis heroic
+		system "Torbab"
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Punisher (Scrappy)"
+	on fail
+		"reputation: Heliarch" = -1000
+		"reputation: Coalition" = -1000
+	on complete
+		"coalition jobs" ++
+		"assisted heliarch" ++
+		payment 800000
+		conversation
+			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, looking at you for a few seconds as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`
+			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, then heads to a Heliarch building.`
+
+
+
+mission "Heliarch License"
+	name "Meet With The Consuls"
+	description "Head to the <destination>, where the Heliarch consuls will make you a Heliarch agent."
+	minor
+	landing
+	source
+		near "Quaru" 1 100
+	destination "Ring of Friendship"
+	to offer
+		or
+			"assisted heliarch" == 5
+			and
+				"coalition jobs" >= 90
+				"assisted heliarch" == 4
+			and
+				"coalition jobs" >= 120
+				"assisted heliarch" == 3
+		not "Lunarium: Questions: done"
+	to fail
+		has "Lunarium: Questions: done"
+	on offer
+		conversation
+			`When you land on <origin>, you find a Heliarch delegation waiting for you outside. When you meet them, the Saryd of the group says, "A great ally to our Coalition, you have proven to be."`
+			`	"Discussed your exploits, the consuls have. Much contentment, your contributions brought us," the Kimek continues.`
```

Comment:
I don't think `contentment` is a good word to use here. Are they more saying you're useful or that they're very happy with you?

Reply:
Changed to satisfaction, does it work better?